pages/resources/quests.mdx

```diff
@@ -0,0 +1,329 @@
+
+###### Example Quest User Status
+
+```json
+{
+  "user_id": "222069018507345921",
+  "quest_id": "8206816794116096000",
+  "enrolled_at": "2077-01-01T11:59:59+00:00",
+  "completed_at": "2077-01-01T11:59:59+00:00",
+  "claimed_at": "2077-01-01T11:59:59+00:00",
+  "claimed_tier": null,
+  "last_stream_heartbeat_at": "2077-01-01T11:59:59+00:00",
+  "stream_progress_seconds": 1800,
+  "dismissed_quest_content": 0,
+  "progress": {}
+}
+```
+
+### Quest Reward Code Object
+
+Represents an object that holds the reward code.
+
+###### Quest Reward Code Structure
+
+| Field      | Type              | Description                                                                                                   |
+| ---------- | ----------------- | ------------------------------------------------------------------------------------------------------------- |
+| quest_id   | snowflake         | The ID of the quest                                                                                           |
+| code       | string            | The redeem code                                                                                               |
+| platform   | string            | The [platform this redeem code applies to](#quest-platform-type)                                              |
+| user_id    | snowflake         | The ID of the user who this code belongs to                                                                   |
+| claimed_at | ISO8601 timestamp | When the user claimed the quest's reward                                                                      |
+| tier       | unknown           | Unused - possibly the same as the `claimed_tier` field in the [user status object](#quest-user-status-object) |
+
+###### Example Quest Reward Code
+
+```json
+{
+  "quest_id": "8206816794116096000",
+  "code": "111-1111111",
+  "platform": 0,
+  "user_id": "222069018507345921",
+  "claimed_at": "2077-01-01T18:41:29.706194+00:00",
+  "tier": null
+}
+```
+
+### Quest Claimed Reward Object
+
+Represents the claimed rewards.
+
+##### Quest Claimed Reward Structure
+
+| Field        | Type                                                       | Description                                                    |
+| ------------ | ---------------------------------------------------------- | -------------------------------------------------------------- |
+| claimed_at   | ISO8601 timestamp                                          | When the reward was claimed                                    |
+| entitlements | array[entitlement object]                                  | The entitlements the user received                             |
+| errors       | array[[JSON error](/topics/opcodes-and-status-codes#json)] | An array of errors that have occured while claiming the reward |
```

Comment:
Are you sure this is an array of standard error objects?

Reply:
That was a guess so I'm not really sure, I'm gonna see what kind of errors it returns in a bit